.browserslistrc

```diff
@@ -0,0 +1 @@
+node >= 12.17
```

Comment:
Because this version unflags `--experimental-modules` thereby enabling support for package.json  [`type`](https://nodejs.org/api/packages.html#type) and [`exports`](https://nodejs.org/api/packages.html#exports) fields by default.